docroot/themes/humsci/humsci_basic/src/scss/components/_pager.scss

```diff
@@ -1,9 +1,32 @@
+// Base pager styles
```

Comment:
are these actually for mini pager? not the base styles?

Reply:
is there a way to write these styles and share styles across the mini pager and full one more?

Reply:
(basically DRY-ing up logic that is used here twice in mini pager and full pager)

Reply:
I removed my comment of `// pager__item styles are written below` as I think it was probably misleading. 

This one was a challenge due to the full pager having classes I could style, but the mini pager doesn't. I was happy to originally get to the point of not using any `!important`. 😆It was pretty gnarly at one point. But I'm glad you called this out as I was able to take a fresh look at it and hopefully make it even better. :tada: